dist/versions-v2.json

```diff
@@ -4,8 +4,8 @@
     "latest": "0.15.15"
   },
   "spago": {
-    "unstable": "0.21.0",
-    "latest": "0.21.0"
+    "unstable": "0.93.28",
+    "latest": "0.93.28"
```

Comment:
Since spago-next is still in alpha, presumably we want to keep “latest” as 0.21.0 (cc @f-f)